libarchive/archive.h

```diff
@@ -155,7 +155,7 @@ __LA_DECL int		archive_version_number(void);
 /*
  * Textual name/version of the library, useful for version displays.
  */
-#define	ARCHIVE_VERSION_ONLY_STRING "3.7.5dev"
+#define	ARCHIVE_VERSION_ONLY_STRING "3.7.5"
```

Comment:
Please don't change this.

Reply:
I don't know what happened here, it's been fixed.